graphix/sim/tensornet.py

```diff
@@ -502,28 +504,30 @@ def get_basis_amplitude(self, basis, **kwagrs):
         coef = self.get_basis_coefficient(basis, **kwagrs)
         return abs(coef) ** 2
 
-    def to_statevector(self, indices=None, **kwagrs):
-        """Retrieve the statevector from the tensornetwork.
-        This method tends to be slow however we plan to parallelize this.
+    def to_statevector(self, skip=False, **kwagrs) -> NDArray:
```

Comment:
Please add very detailed description (100+ words, with equations and diagrams) to this method as this is rather uncommon way to use TN (for most people, at least afaik). 
Is this proven to work for any case or are there limitations? worth pointing out we're NOT performing the intermediate measurements according to probability distribution? what is the expected space/time cost compared to statevector sim?

Reply:
I added that based on notes and discussion in #92 .